### PR TITLE
Upgrade Dependencies

### PR DESCRIPTION
**Pull request by nitro-marky**
This PR:

- Updates dependencies
- Removes Helm charts
- Updates check version and release flow
- Adds c8

### CODE REVIEW

File: .github/workflows/release.yml
Comment (dblane-digicatapult) [inline .github/workflows/release.yml:47]:
```suggestion
     - name: Use npm latest
        run: npm install -g npm@latest
```

File: .github/workflows/tests.yml
Comment (dblane-digicatapult) [inline .github/workflows/tests.yml:131]:
```suggestion
```

Comment (dblane-digicatapult) [reply .github/workflows/tests.yml:131]: We want this job to run in parallel with the others.

File: Dockerfile
Comment (dblane-digicatapult) [inline Dockerfile:1]:
```suggestion
# syntax=docker/dockerfile:1.5
```

File: Dockerfile
Comment (dblane-digicatapult) [inline Dockerfile:3]:
```suggestion
FROM node:lts-alpine AS build
```

File: Dockerfile
Comment (dblane-digicatapult) [inline Dockerfile:24]:
```suggestion
FROM node:lts-alpine AS runtime
```

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml
Comment (dblane-digicatapult) [reply .github/workflows/release.yml:47]: Also remove the underlying `run command`

File: .github/workflows/tests.yml
Comment (dblane-digicatapult) [inline .github/workflows/tests.yml:142]:
```suggestion
      - name: Build multi-arch image
```
We aren't publishing.

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

**Review by dblane-digicatapult — APPROVED** [review]
Looks good to me, wait for @jonmattgray or @n3op2 to review before merging.

**Review by jonmattgray — APPROVED** [review]

**Pull request merged**
